packages/doenetml-worker-rust/lib-doenetml-core/src/core/components/actions.rs

```diff
@@ -0,0 +1,14 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+#[cfg_attr(feature = "web", derive(tsify::Tsify))]
+#[cfg_attr(feature = "web", tsify(from_wasm_abi))]
+pub struct ActionBody<T> {
+    pub args: T,
+}
+
+#[derive(Debug, Serialize, Deserialize)]
+#[cfg_attr(feature = "web", derive(tsify::Tsify))]
+#[cfg_attr(feature = "web", tsify(from_wasm_abi))]
+pub struct ActionBodyEmpty {}
```

Comment:
Do we need `ActionBodyEmpty`? It's not currently used, as you just are using unit variants when there is no action body.